airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -278,7 +278,7 @@ def execute(self, context: 'Context') -> str:
             self._await_status(
                 item_type,
                 self.target_db_snapshot_identifier,
-                wait_statuses=['copying'],
+                wait_statuses=['creating', 'copying'],
                 ok_statuses=['available'],
```

Comment:
I'm approving to unblock the system test work but this seems a bit strange logic?
If the goal is to wait for the ok status then why we care about the wait status?
Shouldn't we just wait till the status is `available`? meaning that anything that isn't the ok_status we just continue to wait.

What am I missing here?

Reply:
This is an operator that hat the option to wait till the action is completed.
To know if it's completed there is a target status (listed as `ok_statuses`)
The need to specify also what are the wait statuses for this scenario seems unnecessary to me.